ci-operator/config/openshift/cluster-monitoring-operator/openshift-cluster-monitoring-operator-master.yaml

```diff
@@ -77,6 +77,13 @@ tests:
   commands: make test-unit
   container:
     from: src
+- as: golangci-lint
+  commands: |
+    export GOCACHE=/tmp/
```

Comment:
I think the env vars are based on `$HOME`, we can just set this?

Reply:
(The closer the command is to what we (may) run locally, the easier it is to debug I think)

Reply:
Take into consideration that this is changing *specific* CI config. In this case, due to the overall prow config we'll be getting into permissions issues unless we do this. As you see, it's basically a cache thing so you'll notice no difference at all when running locally. In fact, you can even use this exact directories locally. I get your point but we wouldn't be able to apply this here.